lib/stat_tracker.rb

```diff
@@ -70,7 +70,7 @@ def average_goals_by_season
     all_games_by_season_id.reduce({}) do |games_by_season, (season, games)|
       total_goals = 0
       games.each do |game|
-        total_goals += game.away_goals.to_f + game.home_goals.to_f
+        total_goals += game.total_goals
```

Comment:
well this turned out way more simple. idk why I didn't come to this conclusion. Good job though